src/app/ui/file_selector.cpp

```diff
@@ -613,38 +613,33 @@ bool FileSelector::show(
     }
     // else file-name specified in the entry is really a file to open...
 
-    // check if the filename doesn't contain slashes or other ilegal characters...
-    bool has_invalid_char = (fn.find('/') != std::string::npos);
 #ifdef _WIN32
-    has_invalid_char =
-      has_invalid_char ||
-      (fn.find('\\') != std::string::npos ||
-       fn.find(':') != std::string::npos ||
-       fn.find('*') != std::string::npos ||
-       fn.find('?') != std::string::npos ||
-       fn.find('\"') != std::string::npos ||
-       fn.find('<') != std::string::npos ||
-       fn.find('>') != std::string::npos ||
-       fn.find('|') != std::string::npos);
-#endif
-    if (has_invalid_char) {
-      const char* invalid_chars =
-        "/"
-#ifdef _WIN32
-        " \\ : * ? \" < > |"
-#endif
-        ;
-
-      ui::Alert::show(
-        fmt::format(
-          Strings::alerts_invalid_chars_in_filename(),
-          invalid_chars));
-
-      // show the window again
-      setVisible(true);
-      goto again;
+    // check if the filename doesn't contain ilegal characters...
+    // Only if it does not exist
+    // Only on Windows, Linux disallows only `/`, and we can consider that just an embedded path if present
+    if (!enter_folder) {
+      bool has_invalid_char =
+         fn.find(':') != std::string::npos ||
```

Comment:
There is a missing `(`, anyway I'll check if I can add support to use full paths on Windows too.
```suggestion
        (fn.find(':') != std::string::npos ||
```